baseplate/lib/tracing.py

```diff
@@ -42,3 +44,72 @@ def should_sample(
 
     def get_description(self) -> str:
         return f"RateLimited(fixed rate sampling {self.rps})"
+
+
+# Greenlet tracing utils
+__Greenlet = gevent.Greenlet
+__IMap = gevent.pool.IMap
+__IMapUnordered = gevent.pool.IMapUnordered
+
+
+class Runnable(Protocol):
+    @property
+    def trace_context(self) -> Context: ...
```

Comment:
```suggestion
    def bp_trace_context(self) -> Context: ...
```

Missed this one I think.

Reply:
Yup, sorry. Just pushed the fix